frontend/src/components/indexPage/MainHeadingContainerMobile.js

```diff
@@ -75,8 +75,7 @@ const getCarouselContent = () => [
   {
     headline: "Climate action platform",
     text: [
-      "Free climate action platform",
-      "bringing everyone involved in climate action together."
+      "Bringing everyone involved in climate action together."
```

Comment:
Updated the text here as "Free climate action platform" seemed redundant from the title. Happy to revert this!

Reply:
Makes sense, good catch!